applications/ecallmgr/src/ecallmgr_originate.erl

```diff
@@ -636,8 +636,10 @@ get_unset_vars(JObj) ->
 -spec maybe_fix_ignore_early_media(kz_term:strings()) -> string().
 maybe_fix_ignore_early_media(Export) ->
     case lists:member("ignore_early_media", Export) of
-        'true' -> "";
-        'false' -> "^unset:ignore_early_media"
+        <<"true">> -> "";
```

Comment:
I don't think this will work; `lists:member/2` will return `boolean()`. Perhaps you want to use `props:get_value/2` instead?